packages/react-percy-webpack/src/entry/index.js

```diff
@@ -1,27 +1,27 @@
+import { EntryNames } from './constants';
 import findSnapshotFiles from './findSnapshotFiles';
+import getEntryPath from './getEntryPath';
 import merge from 'webpack-merge';
-import path from 'path';
+import writeRenderEntry from './writeRenderEntry';
+import writeSnapshotsEntry from './writeSnapshotsEntry';
 
 export default function configureEntry(webpackConfig, percyConfig) {
-  const snapshotFiles = findSnapshotFiles(percyConfig);
+  const frameworkEntryFile = getEntryPath(EntryNames.framework);
 
-  const entry = snapshotFiles.reduce((prev, file) => {
-    const entryName = path
-      .relative(percyConfig.rootDir, file)
-      .replace(new RegExp(`${path.extname(file)}$`), '');
-    return {
-      ...prev,
-      [entryName]: file,
-    };
-  }, {});
+  const renderEntryFile = getEntryPath(EntryNames.render);
+  writeRenderEntry(percyConfig, renderEntryFile);
```

Comment:
We also write a render entry file that loads the renderer the user specified in their config (at this point, basically just the React renderer).